src/main/java/gigaherz/survivalist/ConfigManager.java

```diff
@@ -254,8 +254,9 @@ public int getAxeLevel(ItemStack stack)
     {
         for (Pair<ItemStack, Integer> customAxe : customChoppingAxes)
         {
-            if (ItemStack.areItemsEqual(customAxe.getLeft(), stack))
+            if (customAxe.getLeft().getUnlocalizedName().equalsIgnoreCase(stack.getUnlocalizedName())) {
```

Comment:
That's NOT the way to compare. I'm not at home, but I believe there's a function like ItemStack.areItemsEqualIgnoreDurability